xtable-core/src/test/java/org/apache/xtable/ITConversionController.java

```diff
@@ -142,6 +149,19 @@ private static Stream<Arguments> generateTestParametersForFormatsSyncModesAndPar
     return arguments.stream();
   }
 
+  private static Stream<Arguments> generateTestParametersForUUID() {
+    List<Arguments> arguments = new ArrayList<>();
+    for (SyncMode syncMode : SyncMode.values()) {
+      for (boolean isPartitioned : new boolean[] {true, false}) {
+        // TODO: Add Hudi UUID support later
```

Comment:
Can you create an issue for this in the project so we don't forget?

Reply:
> Can you create an issue for this in the project so we don't forget?

Sure,  I have created an issue for this problem: https://github.com/apache/incubator-xtable/issues/543

Reply:
Can you reference the issue in the comment as well?